test/babelish/test_csv2base.rb

```diff
@@ -10,38 +10,38 @@ def test_initialize
       :excluded_states => excluded_states,
       :state_column => state_column,
       :keys_column => keys_column
-      })
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.